core-bundle/tests/Image/IntegrationTest.php

```diff
@@ -0,0 +1,1688 @@
+            function () {
+                return [
+                    [
+                        $this->getFilesAdapter([
+                            [
+                                'path' => 'files/public/foo.jpg',
+                                'meta' => 'a:1:{s:2:"en";a:3:{s:5:"title";s:9:"foo title";s:3:"alt";s:7:"foo alt";s:7:"caption";s:11:"foo caption";}}',
+                            ],
+                        ], $filesModel),
+                        static function (): void {
+                            $GLOBALS['objPage']->layout = 1;
+                            $GLOBALS['objPage']->layoutId = 1;
+                        },
+                    ],
+                    [new FrontendTemplate('ce_image'),
+                        [
+                            'id' => '1',
+                            'type' => 'image',
+                            'overwriteMeta' => '1 ',
+                            'singleSRC' => 'files/public/foo.jpg',
+                            'alt' => 'bar alt',
+                            'imageTitle' => 'bar title',
+                            'size' => 'a:3:{i:0;s:3:"150";i:1;s:3:"100";i:2;s:4:"crop";}',
+                            'imagemargin' => 'a:5:{s:6:"bottom";s:1:"3";s:4:"left";s:1:"4";s:5:"right";s:1:"2";s:3:"top";s:1:"1";s:4:"unit";s:2:"px";}',
+                            'imageUrl' => '',
+                            'fullsize' => '1',
+                            'caption' => 'bar caption',
+                            'floating' => 'above',
+                            'linkTitle' => '',
+                        ],
+                        null,
+                        null,
+                        $filesModel,
+                    ],
+                ];
+            },
+            array_replace_recursive(
+                $baseExpectedTemplateData,
+                [
+                    'picture' => [
+                        'alt' => 'bar alt',
+                    ],
+                    'lightboxPicture' => [
+                        'img' => [
+                            'src' => 'assets/images/<anything>',
+                            'srcset' => 'assets/images/<anything>',
+                            'hasSingleAspectRatio' => true,
+                            'height' => 30,
+                            'width' => 40,
+                        ],
+                        'sources' => [],
+                    ],
+                    'alt' => 'bar alt',
+                    'imageUrl' => '',
+                    'caption' => 'bar caption',
+                    'linkTitle' => 'bar title',
+                    'href' => 'assets/images/<anything>',
+                    'fullsize' => true,
+                    'attributes' => ' data-lightbox=""',
+                    'floatClass' => ' float_above',
+                    'margin' => 'margin:1px 2px 3px 4px;',
+                ]
+            ),
+        ];
+
+        // todo: add a gallery/text element test case
```

Comment:
This needs to be addressed, too.

Reply:
Turned out this is just a leftover comment from rewriting the functional tests as integration tests.

Removed in d6e47aae0cbbad66c590e24371a271da45f0e2d7.